[2022-02-02 18:49:02,317] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [queued]>
[2022-02-02 18:49:02,334] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [queued]>
[2022-02-02 18:49:02,335] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 18:49:02,338] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-02 18:49:02,340] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-02 18:49:02,360] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-03-01 00:00:00+00:00
[2022-02-02 18:49:02,370] {standard_task_runner.py:52} INFO - Started process 10470 to run task
[2022-02-02 18:49:02,382] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2020-03-01T00:00:00+00:00', '--job-id', '344', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmp5tb2_u0e', '--error-file', '/tmp/tmpxuw7jqm0']
[2022-02-02 18:49:02,387] {standard_task_runner.py:77} INFO - Job 344: Subtask download_dataset_task
[2022-02-02 18:49:02,501] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [running]> on host 700aeb16bae0
[2022-02-02 18:49:02,562] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-02 18:49:02,604] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-03-01T00:00:00+00:00
[2022-02-02 18:49:02,606] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-02 18:49:02,607] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2020-03.csv > /opt/***/yellow_tripdata_2020-03.csv']
[2022-02-02 18:49:02,629] {subprocess.py:85} INFO - Output:
[2022-02-02 18:50:27,245] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-02 18:50:27,285] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20200301T000000, start_date=20220202T184902, end_date=20220202T185027
[2022-02-02 18:50:27,364] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-02 18:50:27,415] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 13:20:35,266] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [queued]>
[2022-02-10 13:20:35,278] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [queued]>
[2022-02-10 13:20:35,279] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 13:20:35,280] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 13:20:35,281] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 13:20:35,293] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-03-01 00:00:00+00:00
[2022-02-10 13:20:35,300] {standard_task_runner.py:52} INFO - Started process 2404 to run task
[2022-02-10 13:20:35,309] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2020-03-01T00:00:00+00:00', '--job-id', '702', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmp6zhknv4_', '--error-file', '/tmp/tmp005yc2jq']
[2022-02-10 13:20:35,312] {standard_task_runner.py:77} INFO - Job 702: Subtask download_dataset_task
[2022-02-10 13:20:35,385] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [running]> on host e30f201fae61
[2022-02-10 13:20:35,428] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 13:20:35,452] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-03-01T00:00:00+00:00
[2022-02-10 13:20:35,454] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 13:20:35,455] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2020-03.csv > /opt/***/yellow_tripdata_2020-03.csv']
[2022-02-10 13:20:35,470] {subprocess.py:85} INFO - Output:
[2022-02-10 13:21:18,091] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 13:21:18,134] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20200301T000000, start_date=20220210T132035, end_date=20220210T132118
[2022-02-10 13:21:18,177] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 13:21:18,239] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-10 15:39:11,198] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [queued]>
[2022-02-10 15:39:11,217] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [queued]>
[2022-02-10 15:39:11,219] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:39:11,221] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 15:39:11,222] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:39:11,239] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-03-01 00:00:00+00:00
[2022-02-10 15:39:11,246] {standard_task_runner.py:52} INFO - Started process 735 to run task
[2022-02-10 15:39:11,252] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2020-03-01T00:00:00+00:00', '--job-id', '821', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpaa09egot', '--error-file', '/tmp/tmpmkvc3x_v']
[2022-02-10 15:39:11,254] {standard_task_runner.py:77} INFO - Job 821: Subtask download_dataset_task
[2022-02-10 15:39:11,346] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [running]> on host 276a0ad90304
[2022-02-10 15:39:11,402] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 15:39:11,435] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-03-01T00:00:00+00:00
[2022-02-10 15:39:11,437] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 15:39:11,440] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2020-03.csv > /opt/***/yellow_tripdata_2020-03.csv']
[2022-02-10 15:39:11,460] {subprocess.py:85} INFO - Output:
[2022-02-10 15:39:39,493] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 15:39:39,539] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20200301T000000, start_date=20220210T153911, end_date=20220210T153939
[2022-02-10 15:39:39,618] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 15:39:39,758] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-10 16:40:13,956] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [queued]>
[2022-02-10 16:40:13,983] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [queued]>
[2022-02-10 16:40:13,986] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:40:13,987] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-10 16:40:13,988] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:40:14,016] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2020-03-01 00:00:00+00:00
[2022-02-10 16:40:14,079] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2020-03-01T00:00:00+00:00', '--job-id', '953', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpmx7l847e', '--error-file', '/tmp/tmptqvq6iw4']
[2022-02-10 16:40:14,096] {standard_task_runner.py:77} INFO - Job 953: Subtask download_dataset_task
[2022-02-10 16:40:14,056] {standard_task_runner.py:52} INFO - Started process 683 to run task
[2022-02-10 16:40:14,320] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2020-03-01T00:00:00+00:00 [running]> on host 106a34111faf
[2022-02-10 16:40:14,438] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 16:40:14,491] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2020-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-03-01T00:00:00+00:00
[2022-02-10 16:40:14,493] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 16:40:14,496] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2020-03.csv > /opt/***/yellow_tripdata_2020-03.csv']
[2022-02-10 16:40:14,535] {subprocess.py:85} INFO - Output:
[2022-02-10 16:40:42,409] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 16:40:42,469] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20200301T000000, start_date=20220210T164013, end_date=20220210T164042
[2022-02-10 16:40:42,547] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 16:40:42,600] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
